sdk/formrecognizer/azure-ai-formrecognizer/samples/README.md

```diff
@@ -0,0 +1,89 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-ai-formrecognizer
+urlFragment: formrecognizer-samples
+---
+
+# Samples for Azure Form Recognizer client library for Python
+
+These code samples show common scenario operations with the Azure Form Recognizer client library.
+The async versions of the samples require Python 3.5 or later.
+
+These sample programs show common scenarios for the Form Recognizer client's offerings.
+
+|**File Name**|**Description**|
+|----------------|-------------|
+|[sample_recognize_custom_forms.py][sample_recognize_custom_forms] and [sample_recognize_custom_forms_async.py][sample_recognize_custom_forms_async]|Recognize forms with your custom model|
+|[sample_recognize_content.py][sample_recognize_content] and [sample_recognize_content_async.py][sample_recognize_content_async]|Recognize text and table structures of a document|
+|[sample_recognize_receipts.py][sample_recognize_receipts] and [sample_recognize_receipts_async.py][sample_recognize_receipts_async]|Recognize data from a file of a US sales receipt using a prebuilt model|
+|[sample_recognize_receipts_from_url.py][sample_recognize_receipts_from_url] and [sample_recognize_receipts_from_url_async.py][sample_recognize_receipts_from_url_async]|Recognize data from a URL of a US sales receipt using a prebuilt model|
+|[sample_train_unlabeled_model.py][sample_train_unlabeled_model] and [sample_train_unlabeled_model_async.py][sample_train_unlabeled_model_async]|Train a custom model with unlabeled data|
+|[sample_train_labeled_model.py][sample_train_labeled_model] and [sample_train_labeled_model_async.py][sample_train_labeled_model_async]|Train a custom model with labeled data|
+|[sample_manage_custom_models.py][sample_manage_custom_models] and [sample_manage_custom_models_async.py][sample_manage_custom_models_async]|Manage the custom models in your account|
+
+## Prerequisites
+* Python 2.7, or 3.5 or later is required to use this package (3.5 or later if using asyncio)
+* You must have an [Azure subscription][azure_subscription] and an
+[Azure Form Recognizer account][azure_form_recognizer_account] to run these samples.
+
+## Setup
+
+1. Install the Azure Form Recognizer client library for Python with [pip][pip]:
+
+```bash
+pip install azure-ai-formrecognizer
+```
+* If authenticating with Azure Active Directory, make sure you have [azure-identity][azure_identity_pip] installed:
```

Comment:
Since FR doesn't currently support AAD, should we have this here?

Reply:
good catch, i'll remove this. thanjks!